Alina Firsova/js.js

```diff
@@ -0,0 +1,194 @@
+var position = 0;
+var order=[];
+
+function remover(element){
+    delIndex=element.parentNode.id;
+   try{ 
+   for (var i = 0; i < order.length; i++){
+        if (order[i].positionInOrder==delIndex){
+            order.splice(i,1);
+            break;
+        }
+   }
+   element.parentNode.remove();
+   orderPriceAndCalorage();
+}
+catch{
+    alert('Редактирование заказа после оплаты невозможно! :(')
+}
+}
+
+
+function add_to_order(element){
+    position=position+1;
+    var check = document.getElementById('check');
+    var addToCheck = document.createElement('div');
+    var foodClass = element.parentNode.id;
+    if (foodClass =='Hamburger'){
+        var size=document.querySelector('input[name=HamburgerSize]:checked').value;
+        var select=document.getElementById('hamburgerStuffing');
+        var stuffing=select.options[select.selectedIndex].value;
+        menuPosition = new Hamburger(size, stuffing, position);
+        order.push(menuPosition);
```

Comment:
@AlinaFirsova You can use index of new added item in array instead of custom `position`. 
And it will be easier to remove item on delete action since it have right index